src/parser.js

```diff
@@ -312,6 +317,10 @@ export default class Parser extends Stream {
         },
         comment() {
           // comments are not important for playback
+          if (!currentUri.tags) {
+            currentUri.tags = [];
+          }
+          currentUri.tags.push(entry.line);
```

Comment:
We decided to put in all tags even if we parse them in case in the future we have to parse a new tag, it would stop appearing in the tags array which could break older code that relied on that tag being in this array before we started parsing it.